### ISSUE TITLE
Segmentation fault (core dumped) Issue

### ISSUE THREAD

**Issue by misbahmunir** [garythung/torch-lrcn#5]
When I run the code on my data set (when I train it on more than 20 videos), it exits with the error - "Segmentation fault (core dumped) " at the validation stage.

There is no issue found at the level of Operating System or package installation at the moment.

**Comment by happsky** [comment]
@misbahmunir have you solved it? I meet the same error.

**Comment by happsky** [comment]
@garythung When I run the code on my data set (27 train videos, 18 val videos, 15 test videos), it exits with the error - "Segmentation fault (core dumped) " at the validation stage.

Here is the error:
=========================================
19:41:46 Getting training data mean frame	
19:41:50 Initializing LRCN	
19:41:54 Starting training for 10 epochs	
19:42:06 Epoch 1 training loss: 1.855981	
19:42:17 Epoch 2 training loss: 1.719661	
19:42:27 Epoch 3 training loss: 1.691099	
19:42:27 Starting loss testing on the val split	
Segmentation fault (core dumped)

**Comment by misbahmunir** [comment]
@happsky Yes, I did. I needed to upgrade the version of Lua. It would run smoothly with Lua 5.2.

**Comment by happsky** [comment]
@misbahmunir I have updated to Lua 5.2.3 via sudo apt-get install lua, but it still not work, can you tell me the way your updated?

**Comment by misbahmunir** [comment]
Here is the reference manual that I used: [http://torch.ch/docs/getting-started.html]
